com.ibm.wala.util/src/main/java/com/ibm/wala/fixpoint/IntSetVariable.java

```diff
@@ -118,6 +120,7 @@ public boolean contains(int b) {
   }
 
   /** @return the value of this variable as a MutableSparseIntSet ... null if the set is empty. */
+  @NullUnmarked /* Annotator://local 1 */
```

Comment:
@nimakarimipour one quick note here is that the `// local` comment is arguably slightly misleading.  It is in fact true that if `@NullUnmarked` is removed, an error will appear on this function since the return is not `@Nullable`.  But, the local fix is easy; probably the real issue is that callers are using the return value without checking if it is null, etc.  I don't know of any kind of easy fix for this; just food for thought.